edit/sections-editor.js

```diff
@@ -46,8 +45,11 @@ function SectionsEditor() {
     },
 
     getEditorTitle(cm) {
-      const index = editor.getEditors().indexOf(cm);
-      return `${t('sectionCode')} ${index + 1}`;
+      const index = editor.getEditors().indexOf(cm) + 1;
+      return {
+        textContent: `#${index}`,
+        title: `${t('sectionCode')} ${index}`,
+      };
```

Comment:
We should also change `getEditorTitle` in `source-editor.js` to return an empty object or null.

Though I think it is better to implement a `getEditorTitleShort` to return a shorthand value of editor title.

Reply:
I agree it's kinda confusing, but I don't like adding methods for trivial stuff, so I've added a typeof check instead.